docs/roadmap.md

```diff
@@ -0,0 +1,59 @@
+# Hermes Roadmap
+
+## MVP
+
+The Minimal Viable Product's goal is to showcase that non-custidial CFD trading on Bitcoin is possible.
+
+## Scope
+
+### Roles
+
+- **Taker**: Cannot create and publish orders, but just takes what is published by a Maker
+- **Maker**: Can create and publish orders
+- **Oracle**: Signs and publishes attested prices at specific points in time
+
+### In scope
+
+For the MVP there is only one Maker that takes the selling side and creates sell orders.
+The maker does not do any automation.
+The maker dictates the price.
+
+A user is always in the role of a taker.
+The user has a simple user interface and can take the maker's order there.
+The taker can specify a quantity, the leverage is fixed to `x5`.
+For the MVP the leverage is fixed to `x5` for both sell and buy orders.
+The oracle is needed for attestation of prices at a certain point in time.
+The oracle is to be run by a separate party that is neither the taker nor the maker.
+
+Overview sequence diagram:
+
+![MVP sequence diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/comit-network/hermes/d12e04d4954deb2ee9ebdc96f5a3ab59b7de370c/docs/asset/mvp_sequence_diagram.puml)
```

Comment:
I updated the diagram. Note that the assumption is that the maker (seller) publishes the contract initially, but the taker (buyer) to close the position by publishing a CET. I think only that makes sense - because as long as the position is open the buyer will have to pay interest. Hence, it is in the buyer's interest to close (=settle on chain atm) the position eventually.

Reply:
From the seller's point of view initially I would not care about closing the position. Eventually this is of course needed. Does that make sense @bonomat ?